app/views/categories/_content_items.html.erb

```diff
@@ -1,7 +1,15 @@
 <ol class="category-detail__list">
-  <% contents.each do |item| %>
+  <% contents.take(6).each do |item| %>
     <li class="category-detail__list-item">
       <%= link_to item.title, item.path %>
     </li>
   <% end %>
 </ol>
+<ol class="collapsible-section category-detail__list category-detail__list--extended js-category-detail__list--extended">
+  <% (contents[6..-1] || []).each do |item| %>
```

Comment:
What are you expecting `contents` to be in the case that it fails over to use the empty array?